src/api/api.ts

```diff
@@ -2,6 +2,9 @@ import { openmrsFetch } from '@openmrs/esm-framework';
 import { careSetting } from '../constants';
 import moment from 'moment';
 
+const BASE_WS_API_URL = '/ws/rest/v1/';
+const BASE_FHIR_API_URL = '/ws/fhir2/R4/';
```

Comment:
`import { fhir } from "@openmrs/esm-framework";

export function performPatientSearch(query: string) {
  return fhir.search({
    type: "Patient",
    query: {
      name: query,
      _getpagesoffset: 0,
      _count: 100,
      _summary: "data"
    }
  });
}
`
@pirupius  you can also use the code above to make fhir calls